src/core/experimental/memory/memcpy.d

```diff
@@ -0,0 +1,401 @@
+/**
+ * Pure D replacement of the C Standard Library memcpy().
+ * There is an idiomatic-D interface, memcpy(), which is split into 3 overloads.
+ * One taking static types, one dynamic arrays and one static arrays.
+ * Also, there is available a C-like interface, the Dmemcpy() (which is named Dmemcpy
+ * for disambiguation with the C memcpy() a _similar_ interface) that
+ * is the classic (void*, void*, size_t) interface.
+ * N.B.: Both the memcpy() here and Dmemcpy() return nothing, contrary to the C Standard
+ * Library version.
+ * Source: $(DRUNTIMESRC core/experimental/memory/memcpy.d)
+ */
+module core.experimental.memory.memcpy;
+
+import core.internal.traits : isArray;
+
+/**
+ * Handle Static Types
+ * N.B.: No need for more sophisticated code for static types. The compiler
```

Comment:
Use full sentences and no abbreviation. Have a look at std.algorithm for good examples on how to write good doc headers.

Reply:
I think it was a note and I was referred to a N.B. So, `Note`?